lib/bloc/blocs/register_bloc.dart

```diff
@@ -32,12 +36,14 @@ class RegisterBloc extends Bloc<RegisterEvent, RegisterState> {
     @required String email,
     @required String password,
     @required String name,
-    String imageURL,
+    File imageURL,
   }) async* {
     yield RegisterState.loading();
     try {
       final String uid = await _userRepository.signUp(email, password);
-      await _accountRepository.createUser(uid, name, email, imageURL);
+      var _result = _storageRepository.uploadUserImage(uid, imageURL);
+      var _imageURL = await (await _result).ref.getDownloadURL();
```

Comment:
.

Reply:
i use _imageURL to distinguish between imageURL param and local _imageURL !